simde/x86/avx512/compress.h

```diff
@@ -0,0 +1,40 @@
+#if !defined(SIMDE_X86_AVX512_COMPRESS_H)
+#define SIMDE_X86_AVX512_COMPRESS_H
+
+#include "types.h"
+
+HEDLEY_DIAGNOSTIC_PUSH
+SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
+SIMDE_BEGIN_DECLS_
+
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m256d
+simde_mm256_mask_compress_pd (simde__m256d src, simde__mmask8 k, simde__m256d a) {
+  #if defined(SIMDE_X86_AVX512VL_NATIVE) && defined(SIMDE_X86_AVX512F_NATIVE)
+    return _mm256_mask_compress_pd(src, k, a);
+  #else
+    simde__m256d_private
+    a_ = simde__m256d_to_private(a),
+    src_ = simde__m256d_to_private(src);
+    size_t j = 0;
```

Comment:
There should be a blank line between the declarations and the next block of code (i.e., between lines 19 and 20) to help make it easy to tell at a glance when the declarations end and the body begins.